docs/README.template.md

```diff
@@ -132,3 +132,27 @@ All configuration in multi-gitter can be done through command line flags, config
 {{end}}{{end}}
 
 Do you have a nice script that might be useful to others? Please create a PR that adds it to the [examples folder](/examples).
+
```

Comment:
@lindell we added a section here in the README to document how to use bitbucket cloud and the limitations as well as links to official documentation.

Reply:
done, comments added about beta support and documented any issues/missing features